quinn-proto/src/connection/mod.rs

```diff
@@ -1591,7 +1591,7 @@ where
         trace!("{:?} keys ready", space);
         if space == SpaceId::Data {
             // Precompute the first key update
-            self.next_crypto = Some(self.crypto.next_1rtt_keys());
+            self.next_crypto = self.crypto.next_1rtt_keys();
```

Comment:
I'd prefer to do `Some(unwrap())` here, to make clear that the crypto layer returning `None` at this stage would violate our invariants.